dss-collection-nft/contracts/DSSCollection.cdc

```diff
@@ -164,7 +166,8 @@ pub contract DSSCollection: NonFungibleToken {
             collectionGroupID: UInt64,
             logicalOperator: String,
             required: Bool,
-            typeName: Type
+            typeName: Type,
+            batchKey: String
```

Comment:
Thanks Jer. `batchKey` is internal technical terms we used in dapper. Should we use `UUID` or `LinkID` to make it more generic?

Reply:
Sure. I have no preference on the name. Will change it in a sec.

Reply:
Changed to metadata: {String: String}